R/format_fy.R

```diff
@@ -0,0 +1,39 @@
+#' Format financial year
+#'
+#' This function formats financial year variables for reporting purposes. It
+#' will convert an year input from 201516 format to 2015-16 format.
+#'
+#' It accepts both numerical and character arguments.
+#'
+#' @param year Financial year
+#' @return Character vector of formatted financial year
+#' @export
+#' @examples
+#' format_fy(201617)
+#' format_fy("201617")
+format_fy <- function(year) {
+  if (!grepl("^[0-9]{6,6}$", year)) {
+    stop("year parameter must be a six digit number e.g. 201617")
```

Comment:
Should we say number or string given it accepts both? (will want to change the AY version too if so)